modules/context-utils-laws/src/main/scala/trace4cats/context/laws/discipline/AskTests.scala

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package io.janstenpickle.trace4cats.base.context
-package laws
-package discipline
+package trace4cats.context.laws.discipline
```

Comment:
Unrelated but might be a good time to address it - it's better for library tests to use a *different* root package from the code under test, to better reflect visibility and implicit resolution for real user code.
```suggestion
package trace4catsTests.context.laws.discipline
```

Reply:
Oh good to know, thanks